test/e2e/host_path.go

```diff
@@ -68,10 +48,10 @@ var _ = Describe("hostDir", func() {
 			fmt.Sprintf("--fs_type=%v", volumePath),
 			fmt.Sprintf("--file_mode=%v", volumePath),
 		}
-		testContainerOutputInNamespace("emptydir r/w on tmpfs", c, pod, 0, []string{
+
+		f.TestContainerOutput("hostdir r/w on tmpfs", pod, 0, []string{
```

Comment:
This test case doesn't really can anything to do with tmpfs.  I suggest calling it 'file mode on hostdir' or something similar.